lesson5/js/main.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const listsMenu = document.getElementById('lists-menu');
+const createdList = document.createElement('ul');
+const menuContents = [{ to: "bookmark.html", img: '1.png', alt: '画像1', text: 'ブックマーク' },
+  { to: 'message.html', img: '2.png', alt: '画像2', text: 'メッセージ' }];
+
+const gatedMenuContents = new Promise((resolve) => {
+  resolve(menuContents);
+});
```

Comment:
IMO
サーバーからデータが返ってくることを想定している & 他で参照されないように

```suggestion
const gatedMenuContents = new Promise((resolve) => {
  resolve( [{ to: "bookmark.html", img: '1.png', alt: '画像1', text: 'ブックマーク' },
  { to: 'message.html', img: '2.png', alt: '画像2', text: 'メッセージ' }]);
});
```

でも良いですけれどね

Reply:
こちらあえて修正しませんでした。

実際このような使い方は少ないのかなと思い、
絶対参照されたくない場合はこのように書くようにします！

ありがとうございます！